package.json

```diff
@@ -3,7 +3,6 @@
   "version": "16.13.0",
   "description": "React package for shallow rendering.",
   "main": "index.js",
-  "module": "esm/index.js",
```

Comment:
this field is nonstandard anyways and shouldn't ever be used; ESM should be shipped via the "exports" field, since that's what all bundlers have to support already anyways.